app/services/etherscan_service.rb

```diff
@@ -13,10 +13,30 @@ def is_proof_member?(address)
       address: address
     )
 
-    if response.dig('result')&.last&.dig('to')&.downcase == address.downcase
+    transactions = response.dig('result')
+
+    inCount = 0
+    outCount = 0
+
+    transactions.each do |child|
+      toAddress = child.dig('to').downcase
+      fromAddress = child.dig('from').downcase
+
+
```

Comment:
tiny nit but can you remove this line... by ruby style guide should be one space here